TCAT/Controllers/RouteDetailViewController.swift

```diff
@@ -236,14 +243,20 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
 
     func getBusLocations() {
 
+// /*
+        
         print("[RouteDetailViewController] getBusLocations")
-
         guard let firstRoute = route.directions.first(where: {
             return $0.routeNumber > 0
         })
-            else { print("[RouteDetailViewController] Couldn't find any valid bus routes"); return }
-
-        Network.getBusLocations(routeID: String(firstRoute.routeNumber)).perform(
+        else {
+            print("[RouteDetailViewController] Couldn't find any valid bus routes")
```

Comment:
add method that printed error message. helps when u look at the console to know where all ur error messages come from. so like `print("[RouteDetailViewController] getBusLocations() Couldn't find any valid bus routes")`